build-tools/build-indexes

```diff
@@ -9,7 +9,7 @@ const rootDir = path.resolve(__dirname, '..');
 process.chdir(rootDir);
 
 if (!fs.existsSync('data/Pokemon-Showdown')) {
-	child_process.execSync('git clone https://github.com/smogon/pokemon-showdown.git', {
+	child_process.execSync('git clone https://github.com/smogon/pokemon-showdown.git Pokemon-Showdown', {
```

Comment:
Good catch! I think my Mac OS is case-insensitive and we haven't re-cloned on the web server in a long time.